IPython/core/display.py

```diff
@@ -622,7 +622,7 @@ class JSON(DisplayObject):
     """
     # wrap data in a property, which warns about passing already-serialized JSON
     _data = None
-    def __init__(self, data=None, url=None, filename=None, expanded=False, metadata=None):
+    def __init__(self, data=None, url=None, filename=None, metadata=None, **kwargs):
```

Comment:
Removing `expanded=False` is a change of API as it's also the 4th positional parameter of the __init__ function.